backend/src/schema/resolvers/users.spec.ts

```diff
@@ -108,25 +109,28 @@ const resetTrophyBadgesSelected = gql`
   }
 `
 
+const database = databaseContext()
```

Comment:
```suggestion
const database = databaseContext()
const { neode } = database
```
So we do not need the following changes in the file

Reply:
I do this deliberately since we want to get rid of neode. This way all calls are to `database.neode` which might be `database.query` or `database.User.create` in the future.